src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -243,6 +243,8 @@ function IOURequestStepScan({
                 const receipt: Receipt = file;
                 receipt.source = source;
                 receipt.state = CONST.IOU.RECEIPT_STATE.SCANREADY;
+                receipt.name = file.name;
```

Comment:
Why do we need to manually assign the `name` property here? Shouldn't the `name` is already there when set `receipt` with `file`?